x-pack/plugin/core/src/main/resources/triggered-watches.json

```diff
@@ -1,21 +1,21 @@
 {
   "index_patterns": [ ".triggered_watches*" ],
-  "order": 2147483647,
-  "settings": {
-    "index.number_of_shards": 1,
-    "index.auto_expand_replicas": "0-1",
-    "index.refresh_interval" : "-1",
-    "index.format": 6,
-    "index.priority": 900
-  },
-  "mappings": {
-    "_doc": {
-      "dynamic" : "strict",
+  "priority": 2147483647,
```

Comment:
Can you add the `_meta` map to this template as well?

Reply:
You're right, I forgot about `_meta` again. I've added it in all templates